src/EnergyPlus/InputProcessing/InputProcessor.cc

```diff
@@ -691,13 +694,13 @@ void InputProcessor::setObjectItemValue(EnergyPlusData &state,
         }
     }
     if (field_type == "a") {
-        if (within_max_fields) NumAlphas++;
+        if (within_max_fields) NumAlphas = alpha_index;
```

Comment:
If the value is being place in the alpha array at `alpha_index` then `NumAlphas` has to match that so that the function calling `getObjectItem` knows about it. If there were unused fields before this, the `NumAlphas` got behind.